### ISSUE TITLE
Accessibility issue - inputGroup has role="presentation"  but child elements have semantic meaning

### ISSUE THREAD

**Issue by emilyuhde** [wojtekmaj/react-time-picker#47]
I set up some accessibility unit tests to test my overrides of the react-time-picker and I started to see this accessibility error. I created a unit test on the react-time-picker straight from the library, with no customizations, and it looks like it's happening on the react-time-picker component. The inputGroup has a role="presentation" set but child elements of it have semantic meaning so that role seems to be inappropriate and is causing the unit test to fail.

(My accessibility unit tests are done with AATT https://github.com/paypal/AATT/blob/master/README.md)

<img width="1240" alt="Screen Shot 2019-07-18 at 11 09 48 AM" src="https://user-images.githubusercontent.com/4451556/61469111-a60d2a00-a94c-11e9-9578-a20b20ebbff3.png">

This element's role is "presentation" but contains child elements with semantic meaning. <div class="react-time-picker__inputGroup" role="presentation" ??? input type="time" name="time" step="60" style="..."

https://www.w3.org/TR/WCAG20-TECHS/F92.html

**Comment by wojtekmaj** [comment]
Hi!
Thanks for the heads up. I wanted to assure you that accessibility is my top concern.

Do you think it would be easy to integrate AATT into our CI tests?

**Comment by wojtekmaj** [comment]
By the way, I ran Lighthouse Accessibility tests on our test suite and fixed all the issues , so we're now aiming for 100 Accessibility score in Lighthouse.

**Comment by emilyuhde** [comment]
I think the actual setup was pretty straight forward when we added AATT tests to our unit testing a few months ago. Once the basic imports, etc, were done, we set up a test util file and that gets imported into the unit test files:

https://gist.github.com/emilyuhde/660260027a9ed18a561dbce567c6fa12

Then we only needed to add one test per component for our component library:

https://gist.github.com/emilyuhde/1f30770860b993f1dad6e5a4430d5ed0

(I keep having issues with code block formatting in Github so I just created some Gists for you.)

**Comment by emilyuhde** [comment]
I did some research into AATT and I found out that the reason why it is reporting this issue but Axe, Wave,  Lighthouse are not is that AATT also uses HTML CodeSniffer. You can add it as a browser bookmark and use it to repro this issue without having to set up the AATT tests.

http://squizlabs.github.io/HTML_CodeSniffer/
